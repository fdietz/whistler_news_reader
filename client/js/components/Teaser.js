import React, { PropTypes } from 'react';
import classNames from 'classnames';

const Teaser = ({ children, className }) => {
  const cls = classNames('teaser', className);
  return (
    <div className={cls}>
      {children}
    </div>
  );
};

Teaser.propTypes = {
  children: PropTypes.node,
  className: PropTypes.string
};

export default Teaser;
