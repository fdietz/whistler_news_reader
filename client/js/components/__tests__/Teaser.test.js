import renderer from 'react-test-renderer';
import React from 'react';
import Teaser from '../Teaser';

describe('Teaser', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Teaser />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
