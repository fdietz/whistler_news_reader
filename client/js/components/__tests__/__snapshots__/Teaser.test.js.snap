exports[`Teaser renders correctly 1`] = `
<div
  className="teaser" />
`;
